client/selectors/main.ts

```diff
@@ -96,6 +96,10 @@ export const getCurrentSortField = createSelector(
     (params) => params.sortField ?? SORT_FIELD.SCHEDULE,
 );
 
+export const getCoverageUser = createSelector(
+    [currentSearch],
+    (params) => params.coverage_user_id ?? 'ALL',
```

Comment:
No need to return `ALL`, `undefined` will suffice (indicating that this param has no value)